frontend/src/routes/GlobalRoutes.tsx

```diff
@@ -1,30 +1,19 @@
-import React, {useEffect, useState} from 'react';
+import React from 'react';
 import Header from "../components/header/Header";
 import RouteMain from "./RouteMain";
 import RouteAuth from "./RouteAuth";
 import {useAppSelector} from "../stores/redux/hooks";
-import {selectLoggedUser} from "../stores/redux/loggedUser/loggedUserSlice";
-import _ from "lodash";
 import Notification from "../components/notification/Notification";
 import Footer from "../components/footer/Footer";
+import {isLogged} from "../stores/redux/loggedUser/loggedUserSlice";
 
 const GlobalRoutes: React.FC = () => {
-
-    const [isLogged, setIsLogged] = useState<boolean>(false);
-    const loggedUser = useAppSelector(selectLoggedUser);
-
-    const updateIsLogged = () => {
-        setIsLogged(!_.isEmpty(loggedUser.token));
-    }
-
-    useEffect(() => {
-        updateIsLogged();
-    }, [loggedUser]);
+    const isUserLogged = useAppSelector(isLogged);
```

Comment:
change name of the method `isLogged` it sounds like a variable but it is not. Moreover, it makes you change your real variables.